build-scripts/xc-universal-er-binary.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
```

Comment:
This script is also ~shamelessly copied from~ based heavily on the work done to compile Glean for the M1. We probably want to push the changes here up into our existing `xc-universal-binary.sh` script, even without any of the new XCFramework/Swift-Package stuff.